Apps/Database/src/Migrations/20201008192612_Dependent.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace HealthGateway.Database.Migrations
+{
+    public partial class Dependent : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Dependent",
+                schema: "gateway",
+                columns: table => new
+                {
+                    ParentHdId = table.Column<string>(maxLength: 52, nullable: false),
+                    DependentHdId = table.Column<string>(maxLength: 52, nullable: false),
+                    CreatedBy = table.Column<string>(maxLength: 60, nullable: false),
+                    CreatedDateTime = table.Column<DateTime>(nullable: false),
+                    UpdatedBy = table.Column<string>(maxLength: 60, nullable: false),
+                    UpdatedDateTime = table.Column<DateTime>(nullable: false),
+                    xmin = table.Column<uint>(type: "xid", nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Dependent", x => new { x.ParentHdId, x.DependentHdId });
```

Comment:
It needs to be composite since there can be many DependentHdId with different parents (many to many)

Reply:
in this case, the data will be dup, why don't we create 2 lookup fields from Dependent to User , ie Guardian1, Guardian2. Or even better to have a Join table called DependentParent (or DependentGuardian) to link Dependent and User in which we can specify the role of a parent (i.e. Father, Mother, Ant, etc...)

Reply:
not sure why it would be duplicate. The pair of hdids will always be unique since the same parent will not have a relationship with the same dependent. I dont think we are tracking the kind of relationship between the users.

This table does not represent the dependent, but the dependent relationship. This is the join table

Reply:
The data will be duplicated if we use an intersection table or if we use a composite key.  The Composite key is more concise in this case.

Reply:
Is it the join table? if so, which one is the primary key of the Dependent table. For me the Dependent table's primary key should be DependentHdId only right?